lib/_http_client.js

```diff
@@ -188,7 +188,6 @@ function ClientRequest(input, options, cb) {
     this.useChunkedEncodingByDefault = true;
   }
 
-  this._ended = false;
```

Comment:
Unfortunately I don't think we can just remove this outright. I've actually seen code in the wild that accessed this :-/ .... we'll have to change it to a deprecated accessor.

Reply:
heh... read ahead James... lol.. I see you added the accessor :-D 

Let's deprecate it tho.